src/app/@theme/components/header/header.component.ts

```diff
@@ -14,17 +9,17 @@ import { filter, map, take } from 'rxjs/operators';
   templateUrl: './header.component.html',
 })
 export class HeaderComponent implements OnInit {
+  isDarkTheme: boolean;
+
   userMenu: NbMenuItem[] = [
     { title: 'Django Admin Interface' },
     { title: 'Log out' },
   ];
-  isDarkTheme: boolean;
 
   constructor(
     private sidebarService: NbSidebarService,
     private nbMenuService: NbMenuService,
-    public userService: UserService,
-    private themeService: NbThemeService
+    public userService: UserService
   ) {
     this.isDarkTheme = HeaderComponent.getThemeName() === 'dark' ? true : false;
```

Comment:
This can be removed in favor of `ThemeSwitcherComponent.getThemeName()`. No point of duplicating the `getThemeName` method in two components.